packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -334,18 +334,36 @@ protected function setUp(): void
                         $columns,
                     ));
 
-                    $example = array_map(
-                        fn (ImportColumn $column) => $column->getExample(),
+                    /*
+                    Сount the number of lines in the file.
+                    It should be equal to the largest number of examples in the columns.
+                    */
+                    $rowsCount = 0;
+                    array_map(
+                        function (ImportColumn $column) use (&$rowsCount) {
+                            $rowsCount = count($column->getExamples()) > $rowsCount ? count($column->getExamples()) : $rowsCount;
+                        },
                         $columns,
                     );
```

Comment:
Please swap this for an `array_reduce()`, where the initial count/carry is 0 and if the number of examples is greater than the carry then it gets returned instead